Packet++/header/DhcpLayer.h

```diff
@@ -416,8 +416,7 @@ namespace pcpp
 		 */
 		IPv4Address getValueAsIpAddr() const
 		{
-			uint32_t addrAsInt = getValueAs<uint32_t>();
-			return IPv4Address(addrAsInt);
+			return getValueAs<uint32_t>();

```

Comment:
why did you remove calling `IPv4Address` c'tor? Is the compiler adding that automatically?

Reply:
Yes. `IPv4Address` has a constructor which takes a `uint32_t` as argument and it will be called automatically.